src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: $cars-width;
+  display: flex;
+  flex-direction: column;
+  background: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+
+  font-family: Roboto, Helvetica, sans-serif;
```

Comment:
you already have this in `html`

Reply:
![1](https://user-images.githubusercontent.com/78701608/204380909-2933849e-a168-4c1a-bd3e-a280a2a24070.jpg)
Strange, when I am adding extra card is working alright.